addons/survey/views/survey_templates.xml

```diff
@@ -330,115 +329,126 @@
             <t t-if="question.question_type == 'simple_choice'"><t t-call="survey.simple_choice"/></t>
             <t t-if="question.question_type == 'multiple_choice'"><t t-call="survey.multiple_choice"/></t>
             <t t-if="question.question_type == 'matrix'"><t t-call="survey.matrix"/></t>
-            <div class="js_errzone alert alert-danger" style="display:none;" role="alert"></div>
+            <div class="o_survey_question_error alert alert-danger" style="display:none;" role="alert"></div>
```

Comment:
`display: none;` -> `d-none` class that you toggle on and off.

Reply:
+1